package.json

```diff
@@ -8,6 +8,7 @@
     "@mapbox/github-file-browser": "0.6.1",
     "bowser": "0.3.2",
     "brfs": "^1.4.3",
+    "buffer": "^5.0.5",
```

Comment:
you don't need to declare buffer as a dependency. as [its readme makes clear](https://github.com/feross/buffer#buffer---), it is automatically included with browserify.